### PR TITLE
[VL] Enable SPARK-10634 timestamp test case

### PR DESCRIPTION
**Pull request by liujiayi771**
## What changes were proposed in this pull request?

Remove exclude for SPARK-10634 timestamp written and read as INT64 - truncation

## How was this patch tested?

SPARK-10634 timestamp written and read as INT64 - truncation

### CODE REVIEW

File: gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala [hunk 1]
```diff
@@ -857,7 +857,6 @@ class VeloxTestSettings extends BackendTestSettings {
     // decimal failed ut
     .exclude("SPARK-34212 Parquet should read decimals correctly")
     // Timestamp is read as INT96.
-    .exclude("SPARK-10634 timestamp written and read as INT64 - truncation")
     .exclude("Migration from INT96 to TIMESTAMP_MICROS timestamp type")
     .exclude("SPARK-10365 timestamp written and read as INT64 - TIMESTAMP_MICROS")
```
Comment (rui-mo) [inline gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:861]: Have you tried above two unit tests, and are they related to the same issue?

File: gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala [hunk 2]
```diff
@@ -857,7 +857,6 @@ class VeloxTestSettings extends BackendTestSettings {
     // decimal failed ut
     .exclude("SPARK-34212 Parquet should read decimals correctly")
     // Timestamp is read as INT96.
```
Comment (rui-mo) [inline gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:859]: This message was added when int64 type was unsupported for timestamp reader. Could you update the message to illustrate why below tests are still disabled?

> .exclude("Migration from INT96 to TIMESTAMP_MICROS timestamp type")
    .exclude("SPARK-10365 timestamp written and read as INT64 - TIMESTAMP_MICROS")

Comment (liujiayi771) [reply gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala:859]: These two unit tests failed because int64 timestamp read is not supported.

File: ep/build-velox/src/get_velox.sh [hunk 3]
```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2024_03_25
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=2024_03_25_ts_fix
```
Comment (rui-mo) [inline ep/build-velox/src/get_velox.sh:20]: Could you revert this change?

**Review by rui-mo — COMMENTED** [review]
Thanks.

**Review by rui-mo — APPROVED** [review]
